plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/ColumnIdentity.java

```diff
@@ -148,9 +148,13 @@ public static ColumnIdentity primitiveColumnIdentity(int id, String name)
     }
```

Comment:
`Co-authored-by: Vikash Kumar <vikash.kumar@starburstdata.com>` 
you can take out this line from the commit comment because you are the one doing the commit 😄